lib/util/experimental-flags.ts

```diff
@@ -0,0 +1,26 @@
+import { GlobalConfig } from '../config/global';
+
+const parsedExperimentalFlags: Record<string, any> = {};
+
+export function experimentalFlagValue(flagName: string): string | null {
+  const experimentalFlags = GlobalConfig.get('experimentalFlags');
+
+  // Check if the flag value is already parsed and stored
+  if (parsedExperimentalFlags[flagName]) {
+    return parsedExperimentalFlags[flagName];
+  }
+
+  if (!experimentalFlags) {
+    return null;
+  }
+
+  for (const flag of experimentalFlags) {
+    if (flag.includes(flagName)) {
+      const [key, value] = flag.split('=');
+      parsedExperimentalFlags[key] = value;
```

Comment:
```suggestion
      parsedExperimentalFlags[key] = value ?? key;
```
otherwise we've undefined in cache and don't get what we want on second call for valueless flags